README.md

```diff
@@ -38,9 +38,26 @@ module "apiary" {
   apiary_log_bucket        = "s3-logs-bucket"
   db_instance_class        = "db.t2.medium"
   db_backup_retention      = "7"
-  apiary_managed_schemas   = ["db1", "db2", "dm"]
+  apiary_managed_schemas   = [
+    {
+        schema_name = "db1",
+        s3_lifecycle_policy_transition_period = "30"
+    },
+    {
+        schema_name = "db_2",
+        s3_storage_class = "INTELLIGENT_TIERING"
+    }
+  ]
   apiary_customer_accounts = ["aws_account_no_1", "aws_account_no_2"]
   ingress_cidr             = ["10.0.0.0/8"]
+  apiary_assume_roles      = [
+    {
+        name = "client_name"
+        principals = [ "arn:aws:iam::account_number:role/cross-account-role" ]
+        schema_names = [ "dm","lz","test_1" ]
+        max_session_duration = "7200"
+    }
+  ]
 }
```

Comment:
If we leave this as two different apiary vars - `producer_iamroles` and `apiary_assume_roles`, somewhere there needs to be some documentation on what the difference is between the two, and the ramifications for one vs. the other.

Also, how do either of these two variables interact with `apiary_external_buckets`?  It looks like from the code that `apiary_external_buckets` have policy of basically allowing Apiary to do anything to those buckets, as long as it is all within the same account.  But are there any cross-account write permissions granted?  Should there be?

Sorry about all the questions, but it is getting hard to keep all this straight and configure this stuff.  I know that I get several questions from other groups any time they try to deploy Apiary.

Reply:
external_buckets are usually managed by other teams, we are not trying to manage any aspect of external buckets other than allowing metastore IAM roles to read & write .

Reply:
ok